qa/1689.out

```diff
@@ -141,11 +159,23 @@ pmproxy.discover.throttled_changed_callbacks PMID: 4.5.18 [filesystem changed ca
 Help:
 number of throttled filesystem change callbacks
 
-pmproxy.pid PMID: 4.1.0 [pmproxy PID]
+pmproxy.mem.datasz PMID: 4.1.5 [server virtual data size]
```

Comment:
Delete all the left-over references to 'server' - they're not adding value to the help text.